packages/tests/integration/src/EntityManager.reactiveRules.test.tsx

```diff
@@ -333,39 +333,53 @@ describe("EntityManager.reactiveRules", () => {
     expect(em2.entities.length).toBe(2);
   });
 
-  it.withCtx("creates the right reactive derived values", async () => {
+  it.withCtx("creates the right reactive field targets", async () => {
     const cstr = expect.any(Function);
-    expect(getMetadata(Book).config.__data.reactiveDerivedValues).toEqual([
-      { kind: "populate", cstr, name: "numberOfBooks", fields: ["author"], path: ["author"] },
-      { kind: "populate", cstr, name: "bookComments", fields: ["author"], path: ["author"] },
-      { kind: "populate", cstr, name: "numberOfPublicReviews", fields: ["author"], path: ["author"] },
-      { kind: "populate", cstr, name: "numberOfPublicReviews2", fields: ["author"], path: ["author"] },
-      { kind: "populate", cstr, name: "tagsOfAllBooks", fields: ["author", "tags"], path: ["author"] },
-      { kind: "populate", cstr, name: "search", fields: ["author", "title"], path: ["author"] },
-      { kind: "populate", cstr, name: "rangeOfBooks", fields: ["author"], path: ["author"] },
-      { kind: "populate", cstr, name: "favoriteBook", fields: ["author"], path: ["author"] },
-      { kind: "populate", cstr, name: "isPublic", fields: ["author"], path: ["reviews"] },
-      { kind: "query", cstr, name: "numberOfBookReviews", fields: ["author"], path: ["author", "publisher"] },
+    expect(getReactiveFields(Book)).toEqual([
+      { kind: "populate", cstr: "Author", name: "numberOfBooks", fields: ["author"], path: ["author"] },
```

Comment:
Most of these changes are changes `cstr: expect.any` -> `cstr: "String"` which won't blow up Jest's diff toString when it crawls the cstr into metadata into all the things. :roll_eyes: